target.c

```diff
@@ -183,8 +195,56 @@ void targetControl(Target *target) {
 						break;
 				}
 			}
+		} else if (target->airCode == CODE_TORMATIC) {
+			for (int i = 0; i < strlen(target->airCommand); i++) {
+				
+				switch (target->airCommand[i]) {
+					case '0':
+					    digitalWrite(target->gpioPin, HIGH);
+						usleep(target->dataLength/3);
+						digitalWrite(target->gpioPin, LOW);
+						usleep(target->dataLength/3*2);
+						printf("0");
+						
+						break;
+
+					case '1':
+					   digitalWrite(target->gpioPin, HIGH);
+						usleep(target->dataLength/3);
+						digitalWrite(target->gpioPin, LOW);
+						usleep(target->dataLength/3);
+						digitalWrite(target->gpioPin, HIGH);
+						usleep(target->dataLength/3);
+						printf("1");
+						break;
+				}
+			}
+			
 		}
+		
+		else if (target->airCode == CODE_WOFI) {
+			
+			usleep(target->sendDelay);
+			
+			for (int i = 0; i < strlen(target->airCommand); i++) {
+				digitalWrite(target->gpioPin, HIGH);
+				switch (target->airCommand[i]) {
+					case '0':
+						
+						usleep(target->dataLength/3.8);
```

Comment:
Hello, 
the data lengths were exactly the ones I could measure. But I just tried it with the values you suggested. This works without any problems. So it can be simplified.

Tormatic: yes, they used to use it so (insecurely) in the past.
Wofi: Sorry I've no Link, But i've replaced a FC-66 remote control with aircontrol.

Reply:
Thanks for the feedback. 👍

Reply:
@memphi2 Can you try controlling your WOFI lamp with the RCO encoding (1)? Should be the same then.